packages/ui-components/src/ClaimButton.ts

```diff
@@ -0,0 +1,688 @@
+      border-bottom: 1px solid #e0e0e0;
+      margin-bottom: 15px;
+      font-size: 14px;
+      font-family: Roboto;
+      width: 100%;
+    }
+    .detail-item {
+      display: flex;
+      align-items: center;
+    }
+    .detail-item:hover {
+      cursor: pointer;
+      background-color: rgba(248, 249, 250, 0.38);
+    }
+    .label {
+      font-weight: bold;
+      color: #2c3e50;
+      margin-right: 5px;
+    }
+    .value {
+      color: #34495e;
+    }
+    .message {
+      margin-top: 10px;
+      font-size: 14px;
+    }
+    .error {
+      color: #dc3545;
+    }
+    .success {
+      color: #28a745;
+    }
+    .tx-link {
+      color: #007bff;
+      text-decoration: underline;
+      cursor: pointer;
+    }
+    h2 {
+      color: #2c3e50;
+      margin-bottom: 10px;
+    }
+    p {
+      color: #7f8c8d;
+      margin-bottom: 15px;
+    }
+    .loader {
+      border: 5px solid #ecf0f1;
+      border-top: 5px solid #3498db;
+      border-radius: 50%;
+      width: 50px;
+      height: 50px;
+      animation: spin 1.5s linear infinite;
+      margin: 20px auto;
+    }
+    @keyframes spin {
+      0% {
+        transform: rotate(0deg);
+      }
+      100% {
+        transform: rotate(360deg);
+      }
+    }
+    #confetti-canvas {
+      width: 100%;
+      height: 200px;
+      margin-top: 10px;
+    }
+    .countdown {
+      font-size: 24px;
+      font-weight: bold;
+      color: #2c3e50;
+      margin-bottom: 12px;
+    }
+    .footer {
+      margin-top: 20px;
+      padding-top: 10px;
+      border-top: 1px solid #e0e0e0;
+      font-size: 12px;
+      color: #7f8c8d;
+      display: flex;
+      justify-content: space-between;
+      align-items: center;
+      flex-direction: column;
+    }
+    .powered-by {
+      display: flex;
+      align-items: center;
+      margin-bottom: 8px;
+    }
+    .gd-link {
+      display: flex;
+      align-items: center;
+      color: #3498db;
+      text-decoration: none;
+      font-weight: bold;
+    }
+    .gd-icon {
+      width: 24px;
+      height: 24px;
+      margin-right: 5px;
+      margin-left: 5px;
+    }
+    .footer-links {
+      display: flex;
+      gap: 10px;
+    }
+    .footer-links a {
+      color: #007bff;
+      text-decoration: underline;
+      cursor: pointer;
+    }
+    .chain-suggestion {
+      cursor: pointer;
+      padding: 8px;
+      border-radius: 6px;
+      background-color: #3498db;
+      color: white;
+      font-weight: bold;
+      box-shadow: 0 2px 4px rgba(0, 0, 0, 0.1);
+    }
+    .chain-suggestion:hover {
+      background-color: #2980b9;
+      cursor: pointer;
+    }
+  `
+
+  firstUpdated() {
+    this.initializeAppKit()
+  }
+
+  async initializeAppKit() {
+    const projectId = "71dd03d057d89d0af68a4c627ec59694"
+    const metadata = {
+      name: "AppKit",
+      description: "AppKit Example",
+      url: "https://example.com",
+      icons: ["https://avatars.githubusercontent.com/u/179229932"],
```

Comment:
this doesnt seem right, should it be our info? if so fill them in
is it supposed to inherit form the integrator app details?

Reply:
You are right, this should be configured through the constructor, will fix